interface/main/dated_reminders/dated_reminders_add.php

```diff
@@ -83,7 +83,7 @@
 // ------- check priority, only allow 1-3
     isset($_POST['priority']) and intval($_POST['priority']) <= 3 and
 // ------- check message, only up to 160 characters limited by Db
-    isset($_POST['message']) and strlen($_POST['message']) <= $max_reminder_words and strlen($_POST['message']) > 0 and
+    isset($_POST['message']) and strlen(utf8_decode($_POST['message'])) <= $max_reminder_words and strlen(utf8_decode($_POST['message'])) > 0 and
```

Comment:
Should this instead be:
mb_strlen
``````
 isset($_POST['message']) and mb_strlen($_POST['message']) <= $max_reminder_words and mb_strlen($_POST['message']) > 0 and
``````

Reply:
Note I am not sure, though. Haven't really used that function before. Interested to hear if that tests well.

Reply:
samething :) but if you wish I can change that to mb_strlen
![image](https://user-images.githubusercontent.com/20621444/37294345-49f4a8d6-261e-11e8-9285-a51ba52b8742.png)